batchflow/named_expr.py

```diff
@@ -656,16 +656,17 @@ class I(NamedExpression):
     Parameters
     ----------
     name : str
```

Comment:
Are substrings needed? Can `name` type be a fixed set of values?

Reply:
This set is too big: `current`/`maximum`/`ratio` or any of the substrings